GetAccept-v2/app.js

```diff
@@ -813,6 +878,18 @@
             return template;
         }
 
+        function templateFolder(templateFolder) {
+            var folder = this;
+            folder.name = templateFolder.name;
+            folder.id = templateFolder.id;
```

Comment:
Should be like this, because getting error `Uncaught SyntaxError: Invalid left-hand side in assignment `